*** rawkit-0.5.0/libraw/bindings.py	2016-01-10 00:13:53.000000000 +0100
--- rawkit-0.5.0-new/libraw/bindings.py	2017-06-26 16:03:07.975784880 +0200
***************
*** 5,8 ****
--- 5,10 ----
  """
  
+ import os
+ 
  from ctypes import *  # noqa
  from ctypes import util
***************
*** 29,33 ****
  
      def __init__(self):  # pragma: no cover
!         libraw = util.find_library('raw')
          try:
              if libraw is not None:
--- 31,38 ----
  
      def __init__(self):  # pragma: no cover
!         #libraw = util.find_library('raw')
!         libraw = os.environ['APPDIRUSR'] + '/lib/libraw.so.15'
!         s = 'LibRaw library: ' + libraw
!         print(s)
          try:
              if libraw is not None:
***************
*** 47,50 ****
--- 52,57 ----
                  'Unsupported Libraw version: %s.%s.%s.' % self.version_number
              )
+         s = 'Libraw version: %s.%s.%s' % self.version_number
+         print(s)
  
          libraw_data_t = structs.libraw_data_t
